profile/models/Takeda/Takeda-unrodded.cpp

```diff
@@ -220,7 +221,7 @@ int main(int argc, char* argv[]) {
   TrackGenerator3D track_generator(&geometry, num_azim, num_polar, azim_spacing,
                                    polar_spacing);
   track_generator.setSegmentFormation(OTF_TRACKS);
-  std::vector<FP_PRECISION> seg_zones {-12.5, 12.5};
+  std::vector<double> seg_zones {-12.5, 12.5};
```

Comment:
Why change this from `FP_PRECISION`?

Reply:
Since seg_zones are part of geometry definition / ray tracing, the input to setSegmentationZones needs to be double precision.